### ISSUE TITLE
`aria-label` generated for github issues raises a warning in the HTML validator

### ISSUE THREAD

**Issue by pchampin** [w3c/respec#3584]
## Important info

- URL to affected spec: https://w3c.github.io/rdf-star/cg-spec/2021-04-13.html
- ReSpec version: 26.5.0
- [x] I did a "hard refresh", but it's still busted.

## Description of problem

**What happened (e.g., it crashed)?:**

The HTML validator complains about the `aria-label` properties generated on included github issues. More precisely, this is the `aria-label` on the github labels of the issue (

See for example validation results here: https://validator.w3.org/nu/?doc=https%3A%2F%2Fw3c.github.io%2Frdf-star%2Fcg-spec%2F2021-04-13.html

**Expected behavior (e.g., it shouldn't crash):**

The validator should pass without any warning.

**Comment by sidvishnoi** [comment]
The `aria-label` should be over links only at least (and not contain the issue title). There are some other issues also (should probably use a `<header>`.
Relevant generated markup:

```html
<div class="issue" id="issue-container-number-3">
  <div role="heading" class="issue-title marker" id="h-issue-1" aria-level="4">
    <a href="https://github.com/w3c/rdf-star/issues/3">
      <span class="issue-number">Issue 3</span>
    </a>
    <span
      class="issue-label"
      aria-label="This issue is labelled as later and process."
    >
      : Do we need more things in the 'conformance' section?
      <a
        class="respec-gh-label"
        style="background-color: rgb(170, 170, 170); color: rgb(0, 0, 0)"
        href="https://github.com/w3c/rdf-star/issues/?q=is%3Aissue+is%3Aopen+label%3A%22later%22"
      >
        later
      </a>
      <a
        class="respec-gh-label"
        style="background-color: rgb(191, 212, 242); color: rgb(0, 0, 0)"
        href="https://github.com/w3c/rdf-star/issues/?q=is%3Aissue+is%3Aopen+label%3A%22process%22"
      >
        process
      </a>
    </span>
  </div>
  <p>For the moment, we only have the boilerplate text generated by respec.</p>
</div>
```

**Comment by marcoscaceres** [comment]
Agree... that aria-label is no good there.

**Comment by marcoscaceres** [comment]
Thanks again for reporting this @pchampin! Moved the-aria label to the anchor. Validator seems happy and hopefully the arial-label makes sense there.
